core-plugins/docs/OracleExport-action.md

```diff
@@ -0,0 +1,94 @@
+# Oracle Export Action
+
+
+Description
+-----------
+A Hydrator Action plugin to efficiently export data from Oracle to HDFS or local file system. The plugin uses
+Oracle's command line tools to export data. The data exported from this tool can then be used in Hydrator pipelines.
+
+
+Use Case
+--------
+A Hydrator user would like to export oracle data onto HDFS or local file system using an action plugin
+that exports Oracle data using the Oracle bulk export tool as opposed to using JDBC.
+
+
+Properties
+----------
+
+**oracleServerHostname:** Hostname of the remote Oracle Host.
```

Comment:
This is not 100% correct. This is **not** the hostname of the Oracle host. This is the hostname of a machine where we will execute `commandToRun` over SSH... It should be possible to skip performing an SSH if this property is blank, meaning the plugin will execute `commandToRun` locally.